components/fxa-client/src/fxa_msg_types.proto

```diff
@@ -72,27 +72,50 @@ message Capabilities {
     repeated Device.Capability capability = 1;
 }
 
-// This is basically an enum with associated values,
-// but it's a bit harder to model in proto2.
-message AccountEvent {
-    enum AccountEventType {
-        TAB_RECEIVED = 1; // data set to TabReceivedData.
+message IncomingDeviceCommand {
+    enum IncomingDeviceCommandType {
+        SEND_TAB = 1; // `data` set to `send_tab_data`.
```

Comment:
Ah, I see you deliberately changed the name from `TAB_RECEIVED` to `SEND_TAB`. I'm happy to keep it this way if it seems better to you (but would be interested in hearing about what motivated it).